contrib/babelfishpg_tsql/src/pl_exec-2.c

```diff
@@ -3658,6 +3661,16 @@ get_insert_bulk_kilobytes_per_batch()
 	return insert_bulk_kilobytes_per_batch;
 }
 
+static const char 
+*gen_schema_name_for_fulltext_index(char *schema_name)
```

Comment:
That is already handled in tsqlIface file, while I’m extracting other data items.

Reply:
Just for sanity, I think schema_name should be declared as const here, e.g. `const char *schema_name`

Reply:
In the query statement, if we have not specified the schema name, it will be an empty string i.e. `""` else we get the proper schema name in the helper function. So the schema name will never be null. Here is the below code for reference - 
```
if (ctx->table_name())
{
	std::string table_info = ::getFullText(ctx->table_name());
	std::string table_name = "";
	std::string schema_name = "";
	size_t pos = table_info.find(".");
	if (pos != std::string::npos) {
		// Extract the schema name before the "."
		schema_name = table_info.substr(0, pos);
		// Extract the table name after the "."
		table_name = table_info.substr(pos + 1);
	} else {
		// No "." character found, set first to the entire string
		table_name = table_info;
	}
	stmt->table_name = pstrdup(downcase_truncate_identifier(table_name.c_str(), table_name.length(), true));
	stmt->schema_name = pstrdup(downcase_truncate_identifier(schema_name.c_str(), schema_name.length(), true));
}
```